### PR TITLE
chore: add qwen1.5-moe-a2.7b and yi-1.5-6b

### PR DESCRIPTION
**Pull request by netwjx**
## Describe Your Changes

- Add Qwen1.5 MoE A2.7B
- Add Yi-1.5 6B

## Self Checklist

- [x] Added Qwen1.5 MoE A2.7B
- [x] Added Yi-1.5 6B

### CODE REVIEW

File: extensions/inference-nitro-extension/resources/models/qwen1.5-moe-a2.7b/model.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  "sources": [
+    {
+      "filename": "Qwen1.5-MoE-A2.7B-Chat.Q4_K_M.gguf",
+      "url": "https://huggingface.co/RichardErkhov/Qwen_-_Qwen1.5-MoE-A2.7B-Chat-gguf/resolve/main/Qwen1.5-MoE-A2.7B-Chat.Q4_K_M.gguf"
+    }
+  ],
+  "id": "qwen1.5-moe-a2.7b",
+  "object": "model",
+  "name": "Qwen1.5 MoE A2.7B Chat Q4",
+  "version": "1.1",
```
Comment (hahuyhoang411) [inline extensions/inference-nitro-extension/resources/models/qwen1.5-moe-a2.7b/model.json:11]: version 1.0

File: extensions/inference-nitro-extension/resources/models/yi-1.5-6b/model.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    ],
+    "id": "yi-1.5-6b",
+    "object": "model",
+    "name": "Yi-1.5 6B Chat Q4",
+    "version": "1.5",
+    "description": "2024-05-13: The Yi-1.5 series models are open-sourced, further improving coding, math, reasoning, and instruction-following abilities.",
+    "format": "gguf",
+    "settings": {
+      "ctx_len": 4096,
+      "prompt_template": "{system_message}<|im_start|>user\n{prompt}<|im_end|>\n<|im_start|>assistant",
```
Comment (hahuyhoang411) [inline extensions/inference-nitro-extension/resources/models/yi-1.5-6b/model.json:16]: missing `<|im_start|>system\n`?

File: extensions/inference-nitro-extension/resources/models/yi-1.5-6b/model.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    "sources": [
+      {
+        "filename": "Yi-1.5-6B-Chat.Q4_K_M.gguf",
+        "url": "https://huggingface.co/MaziyarPanahi/Yi-1.5-6B-Chat-GGUF/resolve/main/Yi-1.5-6B-Chat.Q4_K_M.gguf"
+      }
+    ],
+    "id": "yi-1.5-6b",
+    "object": "model",
+    "name": "Yi-1.5 6B Chat Q4",
+    "version": "1.5",
```
Comment (hahuyhoang411) [inline extensions/inference-nitro-extension/resources/models/yi-1.5-6b/model.json:11]: version 1.0

**Comment by louis-jan** [comment]
What is the status? @hahuyhoang411

**Comment by louis-jan** [comment]
Resolved

**Comment by louis-jan** [comment]
Hi @netwjx, thanks for your contribution. FYI, we are moving models to [Cortexso](https://huggingface.io/cortexso) and will close this PR soon.

**Pull request closed without merging**